src/utils/strings.js

```diff
@@ -42,8 +42,14 @@ export function escapeHtml(str) {
 
 // Add the correct grammatical number (singular or plural)
 // to a noun depending on the number of items of an array
-export function pluralize(array) {
-  return array.length === 1 ? '' : 's';
+export function pluralize(str, array) {
```

Comment:
What you've changed here is an improvement over what was there before, but I am a little concerned about the fact that we're hard-coding in some specialized logic for pluralization that only works for specific English words.

I have an idea about a way to make this more generic and will be easier to update for a future world when we need to support alternate localizations. Maybe I'll create a quick PR against this one and you can let me know what you think.

Reply:
Maybe it's going a little overboard, but I started working on this today, because I think it would be really useful for the hopefully near future where we can provide localized translations of UI elements.

Here's the diff if you're curious about the direction: https://github.com/hqhhuang/swift-docc-render/compare/update-frameworks-to-technologies...mportiz08:generic-pluralize (haven't finished testing yet)

Reply:
New pluralisation looks good to me.

Reply:
https://github.com/hqhhuang/swift-docc-render/pull/1